src/transcoder/transcodedialog.cpp

```diff
@@ -447,7 +472,16 @@ QString TranscodeDialog::GetOutputFileName(const QString &input_filepath, const
   else {
     QString filename = TrimPath(input_filepath);
     filename = filename.section(u'.', 0, -2);
-    output_filepath = destination_path + QLatin1Char('/') + filename + QLatin1Char('.') + preset.extension_;
+    // If checkbox for preserving import directory structure is checked validate the path exists
+    if (ui_->preserve_dir_structure->isChecked()) {
+      QString path_to_validate = destination_path + QLatin1Char('/') + input_import_dir + QLatin1Char('/');
```

Comment:
Should be able to use `u'/'` instead of `QLatin1Char('/')`

Reply:
For sure, I was just following the surrounding style. Will update all the instances.